src/components/message/index.js

```diff
@@ -73,6 +73,7 @@ class Message extends Component<Props> {
       replyToMessage({
         threadId,
         messageId: message.id,
+        snippet: window.getSelection().toString() || null,
```

Comment:
Ohh TIL about `window.getSelection`, that's very neat! My main question about this approach is: how can we make sure that whatever is quoted is actually in the message that's being quoted?

Reply:
I commented about it in issue https://github.com/withspectrum/spectrum/issues/2969#issuecomment-393448330